metron-platform/metron-pcap-backend/src/main/java/org/apache/metron/spout/pcap/deserializer/Deserializers.java

```diff
@@ -16,31 +16,35 @@
  * limitations under the License.
  */
 
-package org.apache.metron.spout.pcap.scheme;
+package org.apache.metron.spout.pcap.deserializer;
 
-import org.apache.storm.spout.MultiScheme;
+import org.apache.metron.common.utils.timestamp.TimestampConverters;
 import org.apache.metron.common.utils.timestamp.TimestampConverter;
-import org.apache.storm.kafka.KeyValueSchemeAsMultiScheme;
 
-public enum TimestampScheme {
-   FROM_KEY( converter -> new KeyValueSchemeAsMultiScheme(new FromKeyScheme().withTimestampConverter(converter)))
-  ,FROM_PACKET(converter -> new FromPacketScheme().withTimestampConverter(converter));
+import java.util.function.Function;
+
+public enum Deserializers {
```

Comment:
It's worth noting here that the new spout does not use multischemes.  We are doing a fair bit of byte munging on data coming into the pcap topology in those multischemes, but the abstraction was clunky.  This makes things a bit clearer.